mmdeploy/backend/base/backend_manager.py

```diff
@@ -89,3 +134,17 @@ def find(self, name: str) -> BaseBackendManager:
 
 
 BACKEND_MANAGERS = BackendManagerRegistry()
+
+
+def get_backend_manager(name: str) -> BaseBackendManager:
+    """Get backend manager.
+
+    Args:
+        name (str): name of the backend.
+    Returns:
+        BaseBackendManager: The backend manager of given name
+    """
+    from enum import Enum
+    if isinstance(name, Enum):
```

Comment:
Is it necessary since the argument `name` is expected to be `str`?

Reply:
For backward compatible